app/src/main/java/phone/vishnu/quotes/activity/MainActivity.java

```diff
@@ -124,13 +127,13 @@ protected void onCreate(final Bundle savedInstanceState) {
         constraintLayout = findViewById(R.id.constraintLayout);
         progressIndicator = findViewById(R.id.mainProgressIndicator);
         chipGroup = findViewById(R.id.homeChipGroup);
-
         initViewPager();
         runInitChecks();
         initFABs();
         initAnimations();
         setUpSearchView();
         setUpChipGroup();
+        swipeListener = new SwipeListener(homeFAB);
```

Comment:
Move to the end of `initFABs()` - I think it goes there, since this deals with init of a listener for FAB.